### ISSUE TITLE
Do not print linebreaks "\n"

### ISSUE THREAD

**Issue by venuselite** [hiro5id/console-log-json#23]
Hi,

I am using this library to forward my logs inside my kubernetes cluster to splunk via fluent-bit

It is failing to process the generated json because of the added linebreaks:
`{"level":"info","message":"Authenticator listening on 4987...","@filename":"app/dist/src/server/index.js","@logCallStack":" Server.<anonymous> (app/dist/src/server/index.js:34:47)\n    at Object.onceWrapper (events.js:420:28)\n    at Server.emit (events.js:314:20)\n    at emitListeningNT (net.js:1350:10)\n    at processTicksAndRejections (internalprocess/task_queues.js:83:21)","@packageName":"authenticator","@timestamp":"2021-04-14T15:10:47.961Z"}`

Is there an option to not have them printed?

Best regards,
Philipp

**Comment by hiro5id** [comment]
I'll take a look.  I just wanted to leave a message to acknowledge your issue.

**Comment by hiro5id** [comment]
For the time being while I fix this, if you wish, you can set the environment variable:
```
CONSOLE_LOG_JSON_NO_STACK_FOR_NON_ERROR="TRUE"
```

To make it so that it does not emit the "@logCallStack" bit that contains all the `\n` characters

**Comment by hiro5id** [comment]
OK @venuselite I have implemented your ask in v2.6.0 by defining the environment variable:
```
CONSOLE_LOG_JSON_NO_NEW_LINE_CHARACTERS="true"
```
it will supress adding any `\n` characters.

If suppressing it via environment variable is not convenient in your scenario please let me know what alternate method you wish to use.

**Comment by venuselite** [comment]
Hi @hiro5id , wow thank you so much for the fast support. It works perfectly and looks amazing! :)
Thank you, have a great day
